canonical/ubuntu.com
# Add new section and code snippet changes

## Done

- Apply copy updates to /landscape/install
- Copy docs https://docs.google.com/document/d/1tU1z9PUm_kSwGJEsopJibE4gI8Gfv_3Dw_qN2_io_ao/edit

## QA

- Go to /landscape/install
- Check that new section and code snippet changes match copy doc request

## Issue / Card

Fixes [WD-13633](https://warthogs.atlassian.net/browse/WD-13633)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
